frontend/src/pages/notebookController/NotebookController.tsx

```diff
@@ -84,66 +85,68 @@ export const NotebookController: React.FC = React.memo(() => {
   };
 
   return (
-    <NotebookControllerContext.Provider
-      value={{
-        images,
-        dashboardConfig,
-        notebook,
-        isNotebookRunning,
-        projectName,
-      }}
-    >
-      <ApplicationsPage
-        title={!isNotebookRunning ? 'Start a Notebook server' : 'Notebook server control panel'}
-        description={!isNotebookRunning ? 'Select options for your Notebook server.' : null}
-        loaded={loaded}
-        loadError={loadError}
-        empty={!isNotebookRunning}
-        emptyStatePage={
-          <SpawnerPage
-            setStartModalShown={setStartShown}
-            updateNotebook={updateNotebook}
-            setNotebookPollInterval={setNotebookPollInterval}
-          />
-        }
+    <QuickStarts>
```

Comment:
Why not remove all `QuickStarts` instances that are on pages and just put it wrapping `<Routes />` in `App.tsx`? We want QuickStarts for everything, no?

Reply:
This is a good question. I didn't see the quick start on `Explore` page for now. So if you navigate from `Enabled` to `Explore`, it also gets disappeared. So I am not sure whether we want this for everything.

Reply:
Why wouldn't we? It's quick starts, why should it ever disappear before you're ready for it to go away?

Reply:
I guess it's because on the `Explore` page we have another drawer panel to help enable applications like this.
<img width="900" alt="Screen Shot 2022-07-25 at 1 54 32 PM" src="https://user-images.githubusercontent.com/37624318/180842686-f5785d51-5f93-41b9-bc75-1ac145e6b863.png">
If we also enable quickstart on this page, it could be like a double side-drawer. Or we can write some more complex logic to avoid this from happening.

Reply:
Interesting... not the best layout, but if that's the case, we should go with your solution for now.